instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/GcMemoryMetrics.java

```diff
@@ -55,7 +56,7 @@ public void bindTo(MeterRegistry registry) {
     GcMetricsNotificationListener gcNotificationListener =
         new GcMetricsNotificationListener(registry);
 
-    Gauge.builder("jvm.experimental.gc.memory.reclaimed", deltaSum, AtomicLong::get)
+    Gauge.builder(METRIC_NAME, deltaSum, AtomicLong::get)
```

Comment:
as far as I can tell this metric reports deltas. @gvsmirnov would it be better if this metric reported cumulative value?

Reply:
Whoopsie, missed this, sry. @gvsmirnov is my private account, @gsmirnov-splk is my work account.

Looking at the data, it already reports a cumulative value -- starts at 0 and only ever goes up until restart. 🤔 Looking at the code, it reports the cumulative sum of deltas of memory used before&after GC. It is good as it is now.

Reply:
@gsmirnov-splk Sorry didn't remember that you have 2 accounts, for some reason github doesn't autocomplete your handle.
Seems like I misread, it is cumulative not delta.